components/ApiTableFilter.vue

```diff
@@ -30,10 +29,10 @@
         :key="field.name"
         :name="field.name"
         class="cursor-pointer bg-transparent fill-red text-center"
-        value=""
+        :value="field.value"
```

Comment:
Setting `:value` so that values set at initialisation (eg. from local storage) are reflected in UI